modules/xfeatures2d/include/opencv2/xfeatures2d.hpp

```diff
@@ -70,25 +72,27 @@ namespace xfeatures2d
 //! @addtogroup xfeatures2d_experiment
 //! @{
 
-/** @brief Class implementing the FREAK (*Fast Retina Keypoint*) keypoint descriptor, described in @cite AOV12 .
+/** @brief Class implementing the FREAK (*Fast Retina Keypoint*) keypoint
+descriptor, described in @cite AOV12 .
 
-The algorithm propose a novel keypoint descriptor inspired by the human visual system and more
-precisely the retina, coined Fast Retina Key- point (FREAK). A cascade of binary strings is
-computed by efficiently comparing image intensities over a retinal sampling pattern. FREAKs are in
-general faster to compute with lower memory load and also more robust than SIFT, SURF or BRISK.
-They are competitive alternatives to existing keypoints in particular for embedded applications.
+The algorithm propose a novel keypoint descriptor inspired by the human visual
+system and more precisely the retina, coined Fast Retina Key- point (FREAK). A
+cascade of binary strings is computed by efficiently comparing image intensities
+over a retinal sampling pattern. FREAKs are in general faster to compute with
+lower memory load and also more robust than SIFT, SURF or BRISK. They are
+competitive alternatives to existing keypoints in particular for embedded
+applications.
```

Comment:
Need to remove massive unrelated changes from this PR.

Reply:
Is there a .clang format file that is conform with opencv indentation etc.?
Else I can do it by hand.

Reply:
There is no clang format file (anyway it can be used with new files only).

Reply:
ok, that should be done.